### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by johnzheng1975**
Fix issue: error: error parsing STDIN: error converting YAML to JSON: yaml: line 30: did not find expected key

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/policy-enforcement/rate-limit/index.md [hunk 1]
```diff
@@ -269,7 +269,7 @@ spec:
           vhost:
             name: "inbound|http|9080"
              route:
-              action: ANY
+               action: ANY
```
Comment (rvennam) [inline content/en/docs/tasks/policy-enforcement/rate-limit/index.md:272]: I don't think this change is correct. Did you see https://github.com/istio/istio.io/pull/9044 ?

**Review by rvennam — CHANGES_REQUESTED** [review]

File: content/en/docs/tasks/policy-enforcement/rate-limit/index.md (hunk 1 above)
Comment (johnzheng1975) [reply content/en/docs/tasks/policy-enforcement/rate-limit/index.md:272]: Yes, #9044 is correct.

**Comment by rvennam** [comment]
Dup of #9044.

**Pull request closed without merging**